### PR TITLE
close env in ProcessPyEnvironment when receive _CLOSE

### PR DESCRIPTION
**Pull request by witwolf**
#206 

`ParallelPyEnvironment` do not close envs when it is closed,    it may become a problem when envs are wrapped in process, those processed will not be terminated 

This pr overwrites `_worker`  , close env when receive `_CLOSE` (just add one line code)

### CODE REVIEW

**Comment by emailweixu** [comment]
I also suggest to send a PR to tf_agent

File: alf/environments/suite_socialbot.py [hunk 1]
```diff
@@ -40,6 +44,60 @@ def __init__(self, env_constructor, flatten=False):
         super(ProcessPyEnvironment, self).__init__(
             env_constructor, flatten=flatten)
 
+    def _worker(self, conn, env_constructor, flatten=False):
+        """It's a little different with `super()._worker`, it close environment when
```
Comment (emailweixu) [inline alf/environments/suite_socialbot.py]: "it closs" => it closes

Comment (witwolf) [reply alf/environments/suite_socialbot.py]: fixed

**Comment by witwolf** [comment]
> I also suggest to send a PR to tf_agent

Ok, i'll send a PR to `tf_agent`

**Pull request merged**